torch/csrc/jit/python_custom_class.cpp

```diff
@@ -7,10 +7,21 @@ namespace jit {
 struct CustomMethodProxy;
 struct CustomObjectProxy;
 
+py::object ScriptClass::__call__(py::args args, py::kwargs kwargs) {
+  auto instance = at::ivalue::Object::create(class_type_, /*numSlots=*/1);
+  script::Method init_method(
+      instance, class_type_.type_->getMethod("__init__"));
+  invokeScriptMethodFromPython(init_method, std::move(args), std::move(kwargs));
```

Comment:
Thanks for refactoring, can we put these three lines in a helper function?

`invokeScriptMethodFromPython(Object obj, string method_name, args, kwargs)`